code/modules/reagents/reagents/reagents_medical.dm

```diff
@@ -92,11 +92,13 @@
 	..()
 	if(volume >= 0.2)
 		M.rejuvenate()
+		M?.mind.suiciding = 0
```

Comment:
Applies to both of these, but the parent proc already checks for M existing so instead you'll want to use `M.mind?.suiciding = 0`